src/test/resources/view/UiPartTest/validFileWithFxRoot.fxml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<fx:root type="seedu.address.ui.TestFxmlObject" xmlns="http://javafx.com/javafx"
-            xmlns:fx="http://javafx.com/fxml">
+<fx:root type="ay2021s1_cs2103_w16_3.finesse.ui.TestFxmlObject" xmlns="http://javafx.com/javafx"
+           xmlns:fx="http://javafx.com/fxml">
```

Comment:
Indentation still appears to be slightly off here compared to the previous state
```suggestion
            xmlns:fx="http://javafx.com/fxml">
```